actors/builtin/paych/paych_test.go

```diff
@@ -87,6 +81,20 @@ func TestPaymentChannelActor_Constructor(t *testing.T) {
 		})
 	}
 
+	t.Run("fails if addr is not resolvable to ID address", func(t *testing.T) {
+		rt := mock.NewBuilder(ctx, paychAddr).
+			WithCaller(callerAddr, builtin.InitActorCodeID).Build(t)
+
+		nonIdAddr := tutil.NewBLSAddr(t, 501)
+
+		rt.ExpectSend(nonIdAddr, builtin.MethodSend, nil, abi.NewTokenAmount(0), nil, exitcode.Ok)
+		rt.ExpectValidateCallerType(builtin.InitActorCodeID)
+		rt.ExpectAbort(exitcode.ErrIllegalArgument, func() {
+			rt.Call(actor.Constructor, &ConstructorParams{To: nonIdAddr})
+		})
+		rt.Verify()
+	})
+
```

Comment:
Please add a test that exercises the resolution of a non-id to ID address.

Reply:
This is done.